### DISCUSSION TITLE
Accessing a scratch org

### DISCUSSION THREAD

**Discussion by thomas-partridge-cirrico** [SFDO-Community-Sprints/Membership-Essentials-App#146] [Q&A]
Hi all, apologies, complete Git noob here. I'm working on a Nonprofit Cloud project that requires a Membership data model and wanted to check out the work that you have been doing.

On the [solutioning page](https://github.com/SFDO-Community-Sprints/MembershipSchemaAndBenefits/wiki/Solutioning) I've found the Metecho scratch org link but when I click it I get "That page cannot be found. Try the home page" ([see screenshot](https://share.zight.com/mXuz8D1Z)).

When I click through to the home page link I see "We couldn't find any projects you have access to on GitHub" ([see screenshot](https://share.zight.com/RBukKr49)). Is this expected? 

Apologies if so, keep up the great work

**Comment by coriobriensfdo** [comment]
Thanks for your interest! The Metecho tool allows for contributing team members (with a certain base-line permission) to create a scratch org, but not the general public at this time. The team is still in the build process for the Memberships package and Metecho is used for team members to build and test the current work in progress. It's not designed to be a demo type tool for an in-flight project at this point. 

My recommendation would be to join the Membership Schema group on the Trailblazer Community so you can stay in the loop on when they plan to launch. :)

**Comment by thomas-partridge-cirrico** [comment]
Ah gotcha. Thanks so much for the quick response. I'll keep an eye out for
updates, keep up the great work :)

Thomas

On Fri, 22 Dec 2023 at 16:09, coriobriensfdo ***@***.***>
wrote:

> Thanks for your interest! The Metecho tool allows for contributing team
> members (with a certain base-line permission) to create a scratch org, but
> not the general public at this time. The team is still in the build process
> for the Memberships package and Metecho is used for team members to build
> and test the current work in progress. It's not designed to be a demo type
> tool for an in-flight project at this point.
>
> My recommendation would be to join the Membership Schema group on the
> Trailblazer Community so you can stay in the loop on when they plan to
> launch. :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SFDO-Community-Sprints/MembershipSchemaAndBenefits/discussions/146#discussioncomment-7929150>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZJJCVW6QJOUNJLAQZHC2STYKWWD7AVCNFSM6AAAAABA7PE4PGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TSMRZGE2TA>
> .
> You are receiving this because you authored the thread.Message ID:
> <SFDO-Community-Sprints/MembershipSchemaAndBenefits/repo-discussions/146/comments/7929150
> @github.com>
>


-- 
*Thomas Partridge*
Senior Salesforce Consultant & Associate Manager · Cirrico
<https://cirrico.com/>
020 7526 9201
199 Bishopsgate, London, EC2M 3TY

Salesforce <http://www.salesforce.com/uk/?ir=1> | My Credentials
<https://trailhead.salesforce.com/credentials/certification-detail-print?searchString=rLP9AzC6Bx0FAR+cyppOCgkoFAL/hxbbh8GHSxpYu1NInvbPPqQ0LCVOeE4m6bhx>
